src/pages/developers/evm/gas.mdx

```diff
@@ -33,7 +33,7 @@ When withdrawing ZRC-20 tokens back to a connected connected chain, the
 "withdraw gas fee" is applicable (listed as "Total fee" in the table below).
 
 To find out the fee amount, call the `withdrawGasFee` function on the [ZRC-20
-contract](https://github.com/zeta-chain/protocol-contracts/blob/main/contracts/zevm/ZRC20.sol)
+contract](https://github.com/zeta-chain/protocol-contracts/blob/3ede272fba5b0c28a25778477b02f0639e5877f3/v2/contracts/zevm/ZRC20.sol)
```

Comment:
Can't we keep `main` here?
```suggestion
contract](https://github.com/zeta-chain/protocol-contracts/blob/main/v2/contracts/zevm/ZRC20.sol)
```

Reply:
Sure. Old habit of using permalinks everywhere.